app/src/main/java/com/kickstarter/ApplicationModule.java

```diff
@@ -214,16 +212,8 @@ static RemotePushClientType provideBrazeClient(final @NonNull Build build, final
   @Provides
   @Nonnull
   @Singleton
-  static FeatureFlagClientType provideFeatureFlagClientType(final @NonNull Build build, final @ApplicationContext @NonNull Context context) {
-    final @NonNull FeatureFlagClient ffClient;
-    if (ContextExt.isKSApplication(context)) {
-      ffClient = new FeatureFlagClient(build, FirebaseRemoteConfig.getInstance());
-      ffClient.initialize();
```

Comment:
Initialization block moved to `FirebaseHelper` due to race condition during app launch, Remote config requires FirebaseApp.initializeApp(context) to be called before any fetch/activate call